ui/src/utils/basepath.ts

```diff
@@ -1,8 +1,11 @@
-import {getRootNode} from 'src/utils/nodes'
+import {BASE_PATH} from 'src/shared/constants'
 
 export const getBasepath = () => {
-  const rootNode = getRootNode()
-  return rootNode.getAttribute('data-basepath') || ''
+  if (BASE_PATH === '/') {
+    return ''
+  }
+
+  return BASE_PATH.slice(0, -1)
```

Comment:
webpack.common.ts ensures that this will contain a trailing slash like `/somePath/`